gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -244,7 +244,7 @@ class NFPNodeDriver(driver_base.NodeDriverBase):
         pconst.LOADBALANCER, pconst.FIREWALL, pconst.VPN]
     SUPPORTED_SERVICE_VENDOR_MAPPING = {
         pconst.LOADBALANCER: ["haproxy"],
-        pconst.FIREWALL: ["vyos"],
+        pconst.FIREWALL: ["vyos", "nfp"],
```

Comment:
Makes sense. However, lets do this after summit. Yogesh tested with this change and we have only couple of hours before summit.